templates/elevation/point.html

```diff
@@ -0,0 +1,43 @@
+{% extends 'base.html' %}
+{% block content %}
+<h3>Elevation Point Query</h3>
+<p>Query the ASTER Global Digital Elevation Model (DEM) at a single point specified by latitude and longitude.</p>
+<p>Example: <a href="/elevation/point/65.0628/-146.1627">/elevation/point/65.0628/-146.1627</a></p>
+<h4>Results from the query above will look like this:</h4>
+<pre>
+{
+  "res": "1 kilometer",
+  "title": "ASTER Global Digital Elevation Model",
+  "units": "meters difference from sea level",
+  "z": {
```

Comment:
The point queries return the min/avg/max data nested inside a `z` key with a `elevation_avg_` prefix for each value. The HUC and protected area queries are not nested under a `z` key and do not have an `elevation_avg_` prefix. We should make these consistent. I prefer the latter approach FWIW.